### ISSUE TITLE
New-PSOneQRCodeURI not exported as a function

### ISSUE THREAD

**Issue by j81blog** [TobiasPSP/Modules.QRCodeGenerator#10]
The new function is available (Thank you!), but it's not being exported in the last module
The Alias is exported, but not the function.

```PowerShell
VERBOSE: Populating RepositorySourceLocation property for module QRCodeGenerator.
VERBOSE: Loading module from path 'C:\Program Files\WindowsPowerShell\Modules\QRCodeGenerator\2.4.1\loader.psm1'.
VERBOSE: Exporting function 'New-PSOneQRCodeVCard'.
VERBOSE: Exporting function 'New-PSOneQRCodeGeolocation'.
VERBOSE: Exporting function 'New-PSOneQRCodeWifiAccess'.
VERBOSE: Exporting function 'New-PSOneQRCodeTwitter'.
VERBOSE: Exporting function 'New-PSOneQRCodeText'.
VERBOSE: Exporting function 'New-PSOneQRCode'.
VERBOSE: Exporting alias 'New-QRCodeGeolocation'.
VERBOSE: Exporting alias 'New-QRCodeWifiAccess'.
VERBOSE: Exporting alias 'New-QRCodeTwitter'.
VERBOSE: Exporting alias 'New-QRCodeVCard'.
VERBOSE: Exporting alias 'New-QRCodeText'.
VERBOSE: Exporting alias 'New-QRCodeURI'.
VERBOSE: Importing function 'New-PSOneQRCodeGeolocation'.
VERBOSE: Importing function 'New-PSOneQRCodeText'.
VERBOSE: Importing function 'New-PSOneQRCodeTwitter'.
VERBOSE: Importing function 'New-PSOneQRCodeVCard'.
VERBOSE: Importing function 'New-PSOneQRCodeWifiAccess'.
VERBOSE: Importing alias 'New-QRCodeGeolocation'.
VERBOSE: Importing alias 'New-QRCodeText'.
VERBOSE: Importing alias 'New-QRCodeTwitter'.
VERBOSE: Importing alias 'New-QRCodeURI'.
VERBOSE: Importing alias 'New-QRCodeVCard'.
VERBOSE: Importing alias 'New-QRCodeWifiAccess'.
```

**Comment by T0biii** [comment]
you right this PR: #7 is still open to fix this

**Comment by T0biii** [comment]
this issue can be closed #7 is merged now 2.5.0

**Comment by TobiasPSP** [comment]
Thanks all!
